docs/guides/integration/docker.md

```diff
@@ -20,43 +20,44 @@ $ docker run ghcr.io/astral-sh/uv --help
 
 uv provides a distroless Docker image including the `uv` binary. The following tags are published:
 
-- `uv:latest`
-- `uv:{major}.{minor}.{patch}`, e.g., `uv:0.4.16`
-- `uv:{major}.{minor}`, e.g., `uv:0.4` (the latest patch version)
+- `ghcr.io/astral-sh/uv:latest`
+- `ghcr.io/astral-sh/uv:{major}.{minor}.{patch}`, e.g., `ghcr.io/astral-sh/uv:0.4.16`
+- `ghcr.io/astral-sh/uv:{major}.{minor}`, e.g., `ghcr.io/astral-sh/uv:0.4` (the latest patch
+  version)
 
 In addition, uv publishes the following images:
 
-<!-- prettier-ignore -->
```

Comment:
Yes, we need to keep this or prettier breaks the rendering of the list which requires an indent of four.

Reply:
Okay, I'll re-add it. What requires an indent of four for nested lists, though, out of curiosity?

Reply:
mkdocs, I guess. https://github.com/astral-sh/uv/pull/5819